config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -15,15 +15,16 @@ spec:
       containerImage: quay.io/tripleozedcentos9/openstack-mariadb:current-tripleo
       storageRequest: 500M
   rabbitmq:
-    template:
-      replicas: 1
-      #resources:
-      #  requests:
-      #    cpu: 500m
-      #    memory: 1Gi
-      #  limits:
-      #    cpu: 800m
-      #    memory: 1Gi
+    templates:
+      rabbitmq:
```

Comment:
We can rename it to `default` we just need to do some adaptation. Options:
* a) change the defaults in cinder and nova to use `default` as the default rabbitmq[1][2] instead of `rabbitmq`. 
* b) change openstack-operator code in a way that if only one item is defined in the templates here then that item is automatically passed to the nova and cinder specs to override the default. 
* c) just change the sample file here and explicitly pass `default` to nova and cinder in this sample.

Which way do you prefer? 

[1] https://github.com/openstack-k8s-operators/cinder-operator/blob/9f1a61c289c82bb75302a52a4e842d21698413ef/api/v1beta1/cinder_types.go#L52-L56
[2] https://github.com/openstack-k8s-operators/nova-operator/blob/62de786f2c5b0f4ed0ae1c44196d8b7444d49ea9/api/v1beta1/nova_types.go#L46

Reply:
I was thinking more like option c). But we can do this as a follow up. Lets get this in and perhaps we can file a new Jira to revisit the idea?